diff --git a/dist/client/plugins/index.d.ts b/dist/client/plugins/index.d.ts
index 2c58e8f611d55c2db9fae02b6fe8daf036e41e81..5729498fbcb623b3d712ada1c16f2af739c07ff1 100644
--- a/dist/client/plugins/index.d.ts
+++ b/dist/client/plugins/index.d.ts
@@ -14,6 +14,7 @@ import { Session } from 'inspector';
 import { Passkey, passkey } from '../../plugins/passkey/index.js';
 export { twoFactorClient } from '../../plugins/two-factor/index.js';
 import { magicLink } from '../../plugins/magic-link/index.js';
+export type { MagicLinkopts } from '../../plugins/magic-link/index.js';
 import { phoneNumber } from '../../plugins/phone-number/index.js';
 import { anonymous } from '../../plugins/anonymous/index.js';
 import { admin } from '../../plugins/admin/index.js';
diff --git a/dist/plugins/magic-link/index.d.ts b/dist/plugins/magic-link/index.d.ts
index f67f6d844c1d84f5b3545978d2492e8f644aab45..8ea3bc96dd93f7dbdb3fd94ebe71726883f07500 100644
--- a/dist/plugins/magic-link/index.d.ts
+++ b/dist/plugins/magic-link/index.d.ts
@@ -253,4 +253,4 @@ declare const magicLink: (options: MagicLinkopts) => {
     }[];
 };
 
-export { magicLink };
+export { magicLink, MagicLinkopts };
